custom_components/tahoma/light.py

```diff
@@ -100,14 +105,14 @@ def turn_on(self, **kwargs) -> None:
 
         if ATTR_BRIGHTNESS in kwargs:
             self._brightness = int(float(kwargs[ATTR_BRIGHTNESS]) / 255 * 100)
-            self.apply_action("setIntensity", self._brightness)
+            self.apply_action(COMMAND_SET_INTENSITY, self._brightness)
 
         elif ATTR_EFFECT in kwargs:
             self._effect = kwargs[ATTR_EFFECT]
-            self.apply_action("wink", 100)
+            self.apply_action(COMMAND_WINK, 100)
```

Comment:
```suggestion
            self.apply_action(self._effect, 100)
```

This is something we should have a look at. Is this not something that is passed dynamically? Not sure if self._effect is really needed as well.